src/python/WMCore/BossAir/Plugins/PyCondorPlugin.py

```diff
@@ -976,22 +962,23 @@ def customizePerJob(self, job):
             jdl.append('+DESIRED_CMSDataLocations = "%s"\n' % ','.join(job['inputDatasetLocations']))
 
         # HighIO jobs
-        highIOjob = job.get('highIOjob')
-        if highIOjob:
+        if job.get('highIOjob'):
             jdl.append('+Requestioslots = 1\n')
+        else:
+            jdl.append('+Requestioslots = 0\n')
 
         # Performance estimates
-        if job.get('estimatedJobTime'):
-            jdl.append('+MaxWallTimeMins = %d\n' % int(job['estimatedJobTime'] / 60.0))
         if job.get('estimatedMemoryUsage'):
```

Comment:
Good idea. Any processing/production job that needs more should really define it elsewhere and pass it along with the job.